.semgrep/custom-rules.yml

```diff
@@ -162,7 +162,7 @@ rules:
   message: String formatted before gettext
   patterns:
     - pattern-either:
-        - pattern-regex: |
-            _\(\s*"(.*)".format\(
-        - pattern-regex: |
+        - pattern-regex: |-
+            _\(\s*".*"\s*\.format\(
```

Comment:
Whitespace can appear both before and after the quotes, hence the additional `\s*`. The `\` before the `.` is necessary to treat it as a literal rather than a metacharacter. The use of capture groups (unescaped parentheses) has no effect, so they can be omitted.